reggie/configs/primary_locale_names/county/california.json

```diff
@@ -0,0 +1,60 @@
+[
+  { "name": "Alameda", "id": "1" },
+  { "name": "Orange", "id": "30" },
+  { "name": "Alpine", "id": "2" },
+  { "name": "Placer", "id": "31" },
+  { "name": "Amador", "id": "03" },
```

Comment:
MAKE SURE to remove the leading zero from "03", or the notification emails may be messed up I think